app.js

```diff
@@ -0,0 +1,51 @@
+const taskManager = () => {
+  let todos = [];
+  const $todoInput = document.getElementById("inputTodo");
+  const $addTodoButton = document.getElementById("addTodoButton");
+  const $todoList = document.getElementById("listTodo");
+  const $feedbackMessage = document.getElementById("feedbackMessage");
+  function renderTodos() {
+    // iterate over the todos list, create specific todos, and insert them in the DOM
```

Comment:
Have a look at this suggestion: https://github.com/hhe-wtag/dom-todo/pull/1#discussion_r1795185766
```suggestion
```